examples/mqtt.rs

```diff
@@ -22,13 +22,16 @@ struct Settings {
 
 async fn mqtt_client() {
     // Construct a Minimq client to the broker for publishing requests.
-    let mut mqtt: Minimq<_, _, 256, 1> = Minimq::new(
-        "127.0.0.1".parse().unwrap(),
-        "tester",
+    let mut buffer = [0u8; 1024];
+    let localhost: minimq::embedded_nal::IpAddr = "127.0.0.1".parse().unwrap();
+    let mut mqtt: minimq::Minimq<'_, _, _, minimq::broker::IpBroker> = minimq::Minimq::new(
         Stack::default(),
         StandardClock::default(),
-    )
-    .unwrap();
+        minimq::ConfigBuilder::new(localhost.into(), &mut buffer)
```

Comment:
Why does this take a `&mut [u8, N]` and not a `&mut [u8]`?

Reply:
I'm not sure what you're asking? Do you want to downconvert the buffer using i.e. `&mut buffer[..]`? Not sure what the benefit would be

Reply:
Ah. Here it coerces.
https://github.com/quartiq/minimq/blob/2b466aa1bd7377d59ba58f76172bdd3422d5570c/src/config.rs#L50